IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetry.m

```diff
@@ -33,85 +33,29 @@ - (NSString *)telemetryString
     return [self serializeCurrentTelemetryString];
 }
 
-- (instancetype)initWithTelemetryString:(NSString *)telemetryString error:(NSError **)error
+- (instancetype)initWithTelemetryString:(__unused NSString *)telemetryString error:(__unused NSError **)error
```

Comment:
If we don't need this method, let's remove it from the protocol?

Reply:
I think that's a good idea, I made that change. We can add it back when we decide on an implementation for that scenario.